examples/shell/mbed/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ mbed_set_mbed_target_linker_script(${APP_TARGET})
 
 project(${APP_TARGET})
 
-target_link_libraries(${APP_TARGET} mbed-os chip)
+target_link_libraries(${APP_TARGET} mbed-os mbed-core mbed-events mbed-rtos chip)

```

Comment:
Why we add mbed-core and mbed-rtos libraries separately? Are they not part of mbed-os target?

Reply:
You are right @ATmobica , I changed it.